python/cudf/cudf/core/column/datetime.py

```diff
@@ -108,6 +108,20 @@ def second(self):
     def weekday(self):
         return self.get_dt_field("weekday")
 
+    def to_pandas(self, index=None, **kwargs):
+        # Workaround until following issue is fixed:
+        # https://issues.apache.org/jira/browse/ARROW-9772
+
+        # Pandas supports only `datetime64[ns]`, hence the cast.
+        pd_series = pd.Series(
+            self.astype("datetime64[ns]").to_arrow(), copy=True
```

Comment:
This does a device --> host copy as well as a host --> host copy. Instead of going through Arrow in this situation should we go through numpy or something else where we can allocate mutable memory and share it into Pandas zero copy?

Reply:
Changed this to go through numpy and then to pandas zero copy